include/cgimap/output_writer.hpp

```diff
@@ -1,15 +1,29 @@
 #ifndef OUTPUT_WRITER_HPP
 #define OUTPUT_WRITER_HPP
 
+#include <memory>
 #include <string>
 #include <stdexcept>
-#include <boost/noncopyable.hpp>
+
+// due to compatibility issues in gcc 4.8
+template<typename T, typename... Args>
+std::unique_ptr<T> make_unique(Args&&... args) {
+    return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
+}
```

Comment:
Do we really need to support a six year old compiler?

Reply:
That’s the current default gcc version on Travis CI. I didn’t want to do too many changes at once and bump up gcc in one step.

Reply:
I think it would make more sense to change the distribution in .travis.yml to xenial, and use gcc 5.x. Trusty will be eol in about two months anyway.